dali/python/nvidia/dali/pipeline.py

```diff
@@ -55,6 +57,55 @@ def _get_default_stream_for_array(array):
     else:
         return None
 
+
+def _prep_data_for_feed_input(data, batch_size, layout):
```

Comment:
As I am using the same implementation in debug mode so I separated it from the feed_input.